apps/movies/src/movies.controller.ts

```diff
@@ -44,7 +47,7 @@ export class MoviesController extends BaseController {
       page,
       limit,
       route: url,
-    });
+    }, { page, limit, releaseType, title, genre });
```

Comment:
```suggestion
    }, { releaseType, title, genre });
```
I think there is no need to put page and limit to params, because these 2 values are used in options for pagination